packages/ui/src/theme/overwrites/components/input.ts

```diff
@@ -0,0 +1,32 @@
+export const Input = {
+  sizes: {
+    sm: {
+      field: {
+        padding: '6px 12px',
+        borderRadius: '6px',
+      },
+    },
+    md: {
+      field: {
+        padding: '8px 16px',
+        borderRadius: '6px',
+      },
+    },
+    lg: {
+      field: {
+        padding: '10px 16px',
+        borderRadius: '6px',
+      },
+    },
+  },
+  variants: {
+    outline: {
+      field: {
```

Comment:
You should probably specify border colors for different states as Chakra's defaults are different than our mockups